src/actions/actions.ts

```diff
@@ -43,6 +43,42 @@ const compareKeypressSequence = function (one: string[], two: string[]): boolean
   return true;
 };
 
+const adjustVimSelectionPosition = function(vimState: VimState, start: Position, end: Position): any {
```

Comment:
Also I generally prefer static functions to free-floating ones. When we split this file up into many files, the decision will make our lives easier. 😃